sdk/cognitiveservices/azure-cognitiveservices-language-luis/azure/cognitiveservices/language/luis/authoring/tests/test_apps.py

```diff
@@ -0,0 +1,254 @@
+from azure.cognitiveservices.language.luis.authoring import LUISAuthoringClient
+from baseTest import *
+import unittest
+
+
+class AppsTests(unittest.TestCase):
+
+    @use_client
+    def test_list_applications(self, client: LUISAuthoringClient):
+        app_id = client.apps.add({
+            'name': "Existing LUIS App",
+            'description': "New LUIS App",
+            'culture': "en-us",
+            'domain': "Comics",
+            'usageScenario': "IoT"
+        })
+        list = client.apps.list()
```

Comment:
Don't call variable with basic types, here you replaced the basic type "list" with your own list, that can't be good :)